.github/workflows/trigger-react.yml

```diff
@@ -0,0 +1,25 @@
+name: Trigger REACT Buildkite Pipeline
+
+on:
+  push:
+    branches: [ "main", "7.*", "8.*" ]
```

Comment:
you can parse protected branches via api call too, to avoid a static list

Reply:
++ Let's hold on that for now to avoid the extra gh call.